extensions/BugModal/web/bug_modal.js

```diff
@@ -638,14 +638,19 @@ $(function() {
                         forceFixPosition: true,
                         lookup: function(query, done) {
                             query = query.toLowerCase();
+                            var that = $('#keywords');
```

Comment:
Could you use `let` instead of creating `that` to keep `#keywords` in scope?

Reply:
Also, using `document.querySelect()` instead of `$` will make it easier to migrate away from jQuery in the future.

Reply:
Sure, thanks!  Added the changes.